utils/fasm2frames.py

```diff
@@ -120,6 +121,10 @@ def check_for_pudc_b(set_feature):
             extra_features = fasm.parse_fasm_string(
                 '\n'.join(roi_j['required_features']))
 
+    # Get required extra features for the part
+    required_features = db.get_required_fasm_features(part)
```

Comment:
Have you tested this logic when `get_required_fasm_features()` returns an empty set?

Reply:
Yes, I did. Actually the `test_fasm2frames.py` invokes fasm2frames on a fake db when there are no required features defined.